SETUP.md

```diff
@@ -388,11 +388,20 @@ Wondering where to start?  See our [contribution guidelines](CONTRIBUTING.md) fo
 ---
 ### Bundle Install Tips
 
-#### Apple Silicon (M1) bundle install steps
+#### Apple Silicon (M1) or Intel Mac bundle install steps
 
-On Apple Silicon, additional steps are required to get `bundle install` to work.
+On Apple Silicon/Intel Mac, additional steps are required to get `bundle install` to work.
 
-In Gemfile.lock, replace the two occurrences of libv8 (8.4.255.0) with libv8-node (15.14.0.0). Also update mini_racer to 0.4.0 (from 0.3.1):
+Simply run (if you're having issues only with part of gems in the command - you can run it with just needed gems)
+```
+bundle update libv8 mini_racer
+```
+To fix issues in one line. It will update the Gemfile.lock file for you in the same way as described below.
+
+OR
+
+In Gemfile.lock, replace the two occurrences of libv8 (8.4.255.0) with libv8-node (15.14.0.0).
```

Comment:
Can we also include Dave's advice from [this](https://codedotorg.slack.com/archives/C0T0PNTM3/p1660066243807489) message?